Practice2/src/styles/components/common-component.css

```diff
@@ -0,0 +1,15 @@
+.heading {
+  line-height: 48px;
+  font-size: 44px;
+  text-align: center;
+  color: var(--black-dark);
+  font-weight: 400;
+}
+
+.button {
+  background-color: var(--orange-light);
+  width: 198px;
```

Comment:
The width is flexible. Don't hard code this value.

Reply:
fixed